src/apiCalls.js

```diff
@@ -73,7 +73,11 @@ const deleteGame = (userID, gameID) => {
         "Content-Type": "application/json",
       },
     }
-  )
+  ).then(response => {
+    if(!response.ok) {
+      throw new Error("Looks like something looked wrong.")
```

Comment:
The English student in me wonders if we can come up with a better phrase. Too many "looks". What a funny word... look, look, look....